handbook/engineering.md

```diff
@@ -127,7 +127,17 @@ Documentation on completing the release process can be found
 
 ## Oncall rotation
 
-The oncall engineer is a second-line responder to questions raised by customers and community members. The Community team is responsible for the first response to GitHub issues, pull requests, and Slack messages in the osquery and other public Slacks. The Customer team is responsible for the first response to messages in private customer Slack channels.
+### Responsibilities
+
+The oncall engineer is a second-line responder to questions raised by customers and community members. The community contact (Kathy) is responsible for the first response to GitHub issues, pull requests, and Slack messages in the osquery and other public Slacks. Kathy is also responsible for the first response to messages in private customer Slack channels. We respond within 1 hour (during business hours) for interactions and ask the oncall engineer to promptly address any questions sent their way. At times, the oncall engineer may be asked to take over the first response duties when Kathy is unavailable.
+
+Pull requests may be routed through the oncall engineer for review. For PRs to the Fleet documentation, the content editor (Desmi) will request review from the oncall. For PRs from the community, the community contact will request review from the oncall. In either case, if the oncall has the knowledge and confidence to review, they should go ahead and do so. Otherwise, they should request review from an engineer with the appropriate domain knowledge. It is the oncall's responsibility to ensure that these PRs are moved forward (either by review with feedback or merge).
```

Comment:
```suggestion
Pull requests may be routed through the on-call engineer for review. For PRs to the Fleet documentation, the content editor (Desmi) will request a review from the on-call. For PRs from the community, the community contact will a request review from the on-call. In either case, if the on-call has the knowledge and confidence to review, they should go ahead and do so. Otherwise, they should request a review from an engineer with the appropriate domain knowledge. It is the on-call engineer's responsibility to confirm that these PRs are moved forward (either by review with feedback or merge).
```